src/App.vue

```diff
@@ -203,9 +203,9 @@ export default class App extends Mixins(DateMixin, FilingTemplateMixin, LegalApi
     return sessionStorage.getItem('PAY_API_URL')
   }
 
-  /** The Name Request Number from the route. */
-  private get nrNumber (): string {
-    return this.$route.query.nrNumber as string
+  /** The Temporary Registration Id from the route */
+  private get tempId (): string {
+    return this.$route.query.id as string
```

Comment:
I think you only use this in 1 place. Having it as a getter is cool but may encourage other devs to use it, whereas they should be using TemporaryId. (If I understand correctly.)

Reply:
yeah I was just transplanting logic from the previous code, but I have removed it as it is only used to set the registration Id